docs/topics/modules/interpreting_stack_report.adoc

```diff
@@ -0,0 +1,47 @@
+[#interpreting_stack_report]
+= Interpreting your Stack Report
+
+The Stack Report comprises three components:
```

Comment:
I think the three components should be listed right here (as a bullet list) and link to the subsections.